src/deepsparse/loggers/build_logger.py

```diff
@@ -325,6 +338,52 @@ def possibly_modify_target_identifiers(
     return data_logging_config
 
 
+def metric_functions_from_predefined(
+    metric_functions: List[MetricFunctionConfig], identifier_prefix: str
```

Comment:
let's rename metric functions to `predefined_metric_functions` or something to specify that this function on ly works for those